server/account/account_test.go

```diff
@@ -3,6 +3,7 @@ package account
 import (
```

Comment:
I feel we need more tests here, especially for negative outcomes. Like an SSO user trying to change password of local user, but doesn't have enough permissions to do it etc.

Reply:
working on it

Reply:
added missing tests